swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
+from .item import Item
+class Clothing(Item):
+    def __init__(self, condition=0,age =0):
+        super().__init__(condition= condition, category="Clothing",age=age)
```

Comment:
👍 good idea hardcoding "clothing" here since that will never change if you are creating an instance of Clothing!

also, be careful with your consistency! either put spaces between all parameters or put no spaces between parameters. Same with assignment symbols

```suggestion
    def __init__(self, condition=0, age=0):
        super().__init__(condition=condition, category="Clothing", age=age)
```